src/hmem_ze.c

```diff
@@ -372,6 +368,23 @@ int ze_hmem_get_base_addr(const void *ptr, void **base)
 	return FI_SUCCESS;
 }
 
+int ze_hmem_get_mem_id(const void *ptr, uint64_t *mem_id)
+{
+	ze_result_t ze_ret;
+	ze_memory_allocation_properties_t mem_props = {};
```

Comment:
Not sure if this is fast path or not, but do we need to initialize the entire structure?

Reply:
The call segfaults if the structure is uninitialized. I can try to find out the minimal set of fields that need to be initialized.

Reply:
updated.